hs_core/irods.py

```diff
@@ -14,10 +14,18 @@ class ResourceIRODSMixin(models.Model):
     class Meta:
         abstract = True
 
-    def __home_path(self):
+    @property
+    def irods_home_path(self):
         """ Return the home path for local iRODS resources """
         return settings.IRODS_CWD
```

Comment:
@alvacouch is this ```irods_home_path``` ever used outside this class? I am just wondering whether there is a need to make this a property of the class. ```settings.IORDS_CWD``` appears pretty straightforward to me and can be directly used.